### ISSUE TITLE
Language switcher not working on WhatDoTheyKnow in Safari

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#5484]
In Safari Version 13.0.3 the language switcher shows "Cymraeg" when the site is in English, and the switcher has no effect (clicking "English" doesn't change what's shown). 

I've noticed this myself and have seen another administrator share a screenshot while discussing another issue showing the site in English and the switcher showing "Cymraeg". 

In Firefox I get what I believe is the expected behaviour, the switcher saying "English" on opening, with the site in English, then switching to say "Cymraeg" and the site switching to Welsh if it is changed. 

On a different but connected point there doesn't appear to be a Welsh translation for two major pieces of text on the front page: "Get answers from the government and public sector" and "Make a request for information to a UK public authority: by law, they have to respond"
<img width="1124" alt="Screenshot 2019-12-03 at 00 21 33" src="https://user-images.githubusercontent.com/547695/70006721-1d430380-1566-11ea-9f47-e2340c331faa.png">

**Comment by gbp** [comment]
I can't reproduce this on Safari 13.0.1. Could you confirm what URL the browser is showing, when choosing the Welsh it should redirect to https://www.whatdotheyknow.com/cy

**Comment by RichardTaylor** [comment]
I too couldn't reproduce this when I first saw it. I first saw it happen a few days ago and was prompted to raise the issue when I saw another admin share a screenshot showing the problem.  I'm not seeing the issue of the word "Cymraeg" being shown while the site is in English now.  

The switcher doesn't work, it lets me switch to Welsh but not back to English.  Gif starts in English, shows a successful switch to Welsh, and then tries to show how attempts to switch back don't work. 

![language-switcher](https://user-images.githubusercontent.com/547695/70054944-7051a080-15d0-11ea-9f64-84ea45c6c5ce.gif)

**Comment by MattK1234** [comment]
Just to note that I've experienced this behaviour before. If I can recreate
it I'll add to the ticket.

Matt

**Comment by gbp** [comment]
Okay from the gif I reckon the /en and /cy URLs are working fine and it just seems to be an issue with the JS. If either of you are able to recreate and are able could you see if there are any errors in the web inspector.

**Comment by MattK1234** [comment]
I noticed this occur today when accessing WhatDoTheyKnow via Chrome on an Android mobile device.

This first happened via an incognito tab which I couldn't screenshot, but when I opened WhatDoTheyKnow via a normal Chrome tab, the same occurred.

This first screenshot shows me initially accessing WDTK, with Cymraeg displayed in the menu although I was on the English version.
![Screenshot_20191205-012159](https://user-images.githubusercontent.com/47503358/70195827-b3aa2d00-16fe-11ea-8d8a-b1a56817abfd.png)

I then click the English button, and English is now displayed in the menu. I'm still on the English site.
![Screenshot_20191205-012220](https://user-images.githubusercontent.com/47503358/70195830-b9a00e00-16fe-11ea-9d3a-ca64ef74d231.png)

I then click the Cymraeg button and end up on the Welsh site, although English is displayed in the menu.
![Screenshot_20191205-012234](https://user-images.githubusercontent.com/47503358/70195837-bdcc2b80-16fe-11ea-9ff5-1ff81c43913d.png)

Clicking English in the above screenshot just opens and closes the sub-menu. If I click Cymraeg I stay on the Welsh site, with Cymraeg displayed in the menu.
![Screenshot_20191205-012245](https://user-images.githubusercontent.com/47503358/70195842-c0c71c00-16fe-11ea-9160-f524c4ca8a2e.png)

Couldn't get Web Inspector as I'm on Chrome.

When I've noticed this before it would be on Chrome or MS Edge, I don't use Safari anywhere.
